test/webhook-apicoverage/tools/tools.go

```diff
@@ -186,3 +191,56 @@ func GetAndWriteTotalCoverage(webhookIP string, outputFile string) error {
 
 	return nil
 }
+
+// GetResourcePercentages calls resource percentage coverage API to retrieve
+// percentage values.
+func GetResourcePercentages(webhookIP string) (
+	*coveragecalculator.CoveragePercentages, error) {
+	client := &http.Client{Transport: &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	},
+	}
+
+	resp, err := client.Get(fmt.Sprintf(WebhookResourcePercentageCoverageEndPoint,
+		webhookIP))
+	if err != nil {
+		return nil, fmt.Errorf(
```

Comment:
`errors.Wrap` is probably suitable here.
Same below.

Reply:
Fixed.